### ISSUE TITLE
403 errors occurring against /ims/api/incident_reports endpoint with invalid event IDs

### ISSUE THREAD

**Issue by wsanchez** [burningmantech/ranger-ims-server#85]
403 errors occurring against /ims/api/incident_reports endpoint with invalid event IDs

Example:
```
2016-07-26T10:48:23-0700 [twisted.python.log#info] "127.0.0.1" Easy E - [26/Jul/2016:17:48:22 +0000] "GET /ims/api/incident_reports/?event=[object%20ProgressEvent];incident=null HTTP/1.0" 403 17 "https://khaki-shack.prettyok.net/ims/events/Scenarios/incidents/new" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36"
```

Note the invalid arguments: `event=[object%20ProgressEvent];incident=null`
Or: `event=[object%20XMLHttpRequestProgressEvent];incident=10`

User agent is Chrome (version 43, 51) in all occurrences of this, so probably something about Chrome JavaScript handling here that differs from Safari.

**Comment by wsanchez** [comment]
OK I can reproduce this with Chrome on my Mac.

**Comment by wsanchez** [comment]
OK, so what's going on here is that Chrome is setting a global variable called `event` which is overriding the variable with the same name that we are setting at page load time.

That seems to happen after an XHR request?  The `event` value is a `ProgressEvent` object that contains some data related to an XHR load of incident data.

**Comment by wsanchez** [comment]
This make me worry that we're going to continue to run into namespace issues and that I should be prefixing all page-global variable names.

**Comment by wsanchez** [comment]
Fixed: 39db3096f04b3121a6c6a2c57e0c3c9bc76307a2
